plopfile.js

```diff
@@ -11,22 +11,56 @@ module.exports = function (plop) {
       // add jsx
       type: 'add',
       path: 'src/ui/{{name}}/index.tsx',
-      templateFile: 'plop-templates/tsx.hbs',
+      templateFile: 'plop-templates/ui/tsx.hbs',
     }, {
       // add scss
       type: 'add',
       path: 'src/ui/{{name}}/index.scss',
-      templateFile: 'plop-templates/scss.hbs',
+      templateFile: 'plop-templates/ui/scss.hbs',
     }, {
       // add test
       type: 'add',
       path: 'src/ui/{{name}}/__tests__/{{name}}.spec.js',
-      templateFile: 'plop-templates/test.hbs',
+      templateFile: 'plop-templates/ui/test.hbs',
     }, {
       // add story
       type: 'add',
       path: 'src/ui/{{name}}/stories/{{name}}.stories.js',
-      templateFile: 'plop-templates/stories.hbs',
+      templateFile: 'plop-templates/ui/stories.hbs',
+    }],
+  });
+
+  plop.setGenerator('basic reducer', {
```

Comment:
On running plop:

<img width="714" alt="Screen Shot 2023-04-11 at 10 17 50 AM" src="https://user-images.githubusercontent.com/11382805/231030382-2a117577-2746-4d0b-aad4-8e1ca90308d8.png">
<img width="831" alt="Screen Shot 2023-04-11 at 10 18 16 AM" src="https://user-images.githubusercontent.com/11382805/231030389-dd848100-2cf6-4539-b3ef-560d91140ce2.png">

Reply:
wow it's a nice tool